doc/source/ndl.rst

```diff
@@ -1,8 +1,30 @@
+Naive Discriminative Learning
```

Comment:
This section is deffinitly missing references about the learning rule and the matrix implementation.

Reply:
I opened an issue for that (#133) to resolve it later